src/types/colonyMotions.ts

```diff
@@ -4,8 +4,20 @@ export enum ColonyMotions {
     both motions and actions we need to have something distinct
   */
   MintTokensMotion = 'MintTokensMotion',
+  CreateDomainMotion = 'CreateDomainMotion',
+  EditDomainMotion = 'EditDomainMotion',
+  SetUserRolesMotion = 'SetUserRolesMotion',
+  MoveFundsMotion = 'MoveFundsMotion',
+  ColonyEditMotion = 'ColonyEditMotion',
+  PaymentMotion = 'PaymentMotion',
 }
 
 export const motionNameMapping = {
   mintTokens: ColonyMotions.MintTokensMotion,
+  addDomain: ColonyMotions.CreateDomainMotion,
+  editDomain: ColonyMotions.EditDomainMotion,
+  setUserRoles: ColonyMotions.SetUserRolesMotion,
+  moveFundsBetweenPots: ColonyMotions.MoveFundsMotion,
+  editColony: ColonyMotions.ColonyEditMotion,
+  makePaymentFundedFromDomain: ColonyMotions.PaymentMotion,
```

Comment:
We need to coordinate better - i've already done it in my PR :)

Reply:
Tbf, at least it's just this small piece of code (and well, the emojis) that should be repeating :P